src/application/tasks.rs

```diff
@@ -64,6 +61,38 @@ impl FromStr for Task {
 
 impl ToString for Task {
     fn to_string(&self) -> String {
-        format!("{}; {}", self.name, self.priority)
+        format!("{}; {}", self.name, self.priority.to_string())
+    }
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+pub enum ESquare {
+    ImportantUrgent = 1,
+    Important = 2,
+    Urgent = 3,
+    Default = 4,
+}
+
+impl ToString for ESquare {
+    fn to_string(&self) -> String {
+        match self {
+            ESquare::ImportantUrgent => String::from("IU"),
+            ESquare::Important => String::from("I"),
+            ESquare::Urgent => String::from("U"),
+            ESquare::Default => String::from(""),
+        }
+    }
+}
+
+impl FromStr for ESquare {
+    type Err = ();
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "UI" | "IU" => Ok(ESquare::ImportantUrgent),
+            "I" => Ok(ESquare::Important),
+            "U" => Ok(ESquare::Urgent),
+            "" => Ok(ESquare::Default),
+            _ => Err(()),
+        }
```

Comment:
I think it would be good idea to first convert string slice into lowercase/uppercase to avoid complications
```suggestion
        match s.to_lowercase() {
            "ui" | "iu" => Ok(ESquare::ImportantUrgent),
            "i" => Ok(ESquare::Important),
            "u" => Ok(ESquare::Urgent),
            "" => Ok(ESquare::Default),
            _ => Err(()),
        }
```